### DISCUSSION TITLE
Enable magewire cache by default

### DISCUSSION THREAD

**Discussion by thijsdewitt** [magewirephp/magewire#93] [General]
When installing Magewire 1.9.x it introduces a new `magewire` cache. This is not enabled by default when installing or updating from a previous version. I noticed in production that often I the magewire call returns 404. I can reproduce this using the Hyva Checkout.

1. Clear all site data and cookies, so you get a new session and have nothing cached.
2. Add a product to your cart, and go to the Hyva Checkout
3. no payment methods are selected yet. I select a payment method, and it's child block will be rendered.
4. Now I click a checkbox in this child block, causing it to make a Magewire call, the response of the magewire call will give a 404 error.

Enabling the `magewire` cache fixes this issue. I suggest enabling the `magewire` cache by default, or if this is not possible, add this to the installation section in the README.md

**Comment by wpoortman** [comment]
Should be solved thanks to @speedupmate MR https://github.com/magewirephp/magewire/pull/100
